modules/core/shared/src/main/scala-2.10/io.circe/TimeInstances.scala

```diff
@@ -0,0 +1,8 @@
+package io.circe
+
+trait TimeEncoders {
```

Comment:
Can we make these traits package-private as well?

Reply:
Also I'd be tempted to name them `JavaTimeEncoder`, etc., but not a big deal either way.

Reply:
The only reason the name was `TimeEncoders` was to match what was in the java8 project, I have no preference personally, I'll make the change. But did you omit the "s" intentionally? The other Encoder traits mixed in `Encoder` all have an "s" at the end (`TupleEncoders`, `ProductEncoders`,  `MidPriorityEncoders`).

Reply:
Thanks, and yes, I meant to include the `s`.